earth_enterprise/src/fusion/config/geconfigureassetroot.cpp

```diff
@@ -388,6 +404,42 @@ void RepairExistingAssetRoot(const AssetRootStatus &status, bool noprompt) {
 }
 
 // ****************************************************************************
+// ***  ListVolumes
+// ****************************************************************************
+void listCurrentVolumes(VolumeDefList *const voldefs, std::vector<std::string> volnames) {
+    printf("\nCurrent Volume Definitions:\n");
```

Comment:
Minor: I could see the use of a flag, such as `-q, --quiet` which would disable these prefix and postfix lines. It may be slightly easier to parse the output of this option without them.